Instruments/microwave_tuning_gui.py

```diff
@@ -200,7 +210,6 @@ def regen_plots(self):
             # {{{ for the last trace, interpolate, and find the min
             if hasattr(self, "interpdata"):
                 xx, yy = self.interpdata
-                dip_frq_GHz = self.dip_frq_GHz
```

Comment:
this was the line causing it to fail flake.  The variable on the left is genuinely never used.